emmet-core/emmet/core/vasp/validation/check_kpoints_kspacing.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+
+def _check_kpoints_kspacing(
```

Comment:
@esoteric-ephemera can you review the code for kpoints checks and let me know if you think it is reasonable? Some of the logic may not be encompass every possible edge-case (not desirable)